src/magnify/apps/core/migrations/0001_initial.py

```diff
@@ -1,4 +1,4 @@
-# Generated by Django 4.0.5 on 2022-07-07 11:34
+# Generated by Django 4.0.6 on 2022-07-10 10:45
```

Comment:
Is it legal to edit an already merged migration ?

Reply:
Of course not but we still havent released it and are not yet in production.